collated/test/A0139552B.md

```diff
@@ -0,0 +1,169 @@
+# A0139552B
+###### \java\guitests\EditCommandTest.java
+``` java
+    @Test
+    public void edit() {    	
+    	
+        TestTask[] currentList = td.getTypicalTasks();
+        assertClearCommandSuccess();
+
+        TestTask aliceTask = new TestTask(td.alice);
+        assertAddSuccess(aliceTask);
+        
+        //test to check each parameter
+        commandBox.runCommand("edit 1 Call Alice from 2pm to 3pm repeat every day -high");
```

Comment:
try not to use magic string; see how the command was built in addressbook and incorporate something similar